CMakeLists.txt

```diff
@@ -540,9 +540,8 @@ include_directories(SYSTEM velox/external)
 if(NOT VELOX_DISABLE_GOOGLETEST)
   set(gtest_SOURCE AUTO)
   resolve_dependency(gtest)
-  set(VELOX_GTEST_INCUDE_DIR
-      "${gtest_SOURCE_DIR}/googletest/include"
-      PARENT_SCOPE)
+  set(VELOX_GTEST_INCLUDE_DIR
```

Comment:
I think the reason its set is because potentially it can be called via Presto native , and thus a scope above can exist ? Would you be able to test your changes out on Presto native and validate it works  : https://github.com/prestodb/presto/blob/master/presto-native-execution/CMakeLists.txt

Reply:
Yes, this is only for presto. I would prefer to remove it entirely but that would require presto to adapt the cmake. Maybe getting the include dir from the gflags::gflags target?

(The typo also exists there ^^ funny noone caught that)

Reply:
Ah, I see. Yes, it sounds like the right fix is to update Presto and then remove this from Velox's CMakeLists.txt.